db/migrate/20211216160348_create_pending_institution_invites.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePendingInstitutionInvites < ActiveRecord::Migration
+  def change
+    create_table :pending_institution_invites do |t|
```

Comment:
Do we need a new table for pending institutions? 
Can we use the current `institutions` table and add a new column to represent the invitation state? (for example `invitation_token` , `invitation_created_at`, invitation_sent_at`, etc. in the `users` table)

Reply:
I have mixed feelings about that. On one side I don't like duplicating a table, its columns, validations, but on the other side I like having a distinct place to store pending data (so we don't have to filter institutions _by default_ => bugs), we can have the columns that we need, can have a distinct set of validations, so...?